07LessonSeven/homework/index.html

```diff
@@ -5,6 +5,34 @@
   <link rel="stylesheet" href="css/style.css">
 </head>
 <body>
+<header>
```

Comment:
Everything in the body needs to be indented over 2-4 spaces. This helps with code readability.